app/models/v2/profile.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module V2
+  # Model for Canonical Profile
+  class Profile < ApplicationRecord
+    self.table_name = 'canonical_profiles'
+
+    sortable_by :title
+
+    scoped_search on: :title, only_explicit: true, operators: %i[like unlike eq ne in notin]
+    scoped_search on: :ref_id, only_explicit: true, operators: %i[eq ne in notin]
+
+    belongs_to :security_guide, class_name: 'V2::SecurityGuide'
+
+    def self.count_by
```

Comment:
Same here. a `FIXME` is needed that after we swap tables this has to be removed.